core/src/main/resources/schema/infinispan-config-10.0.xsd

```diff
@@ -535,8 +535,12 @@
     <xs:sequence>
       <xs:element name="advanced-externalizer" minOccurs="0" maxOccurs="unbounded">
         <xs:annotation>
-          <xs:documentation>
-            AdvancedExternalizer provides an alternative way to provide externalizers for marshalling/unmarshalling user defined classes that overcome the deficiencies of the more user-friendly externalizer definition  model explained in Externalizer.
+          <xs:documentation>Deprecated since 10.0. Please utilise ProtoStream based marshalling for your Pojos by
```

Comment:
I'm ok with everything except the word Pojo. Can we just say Java objects?

Reply:
Updated